backend/tomato/api/account.py

```diff
@@ -14,6 +14,7 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>
+from imaplib import Flags
```

Comment:
I have no idea why this even worked when I tested the code.
Also, in Eclipse, Ctrl-clicking this brings me to the implementation I want.

fixed it.